erpnext_documentation/www/docs/user/manual/en/customize-erpnext/articles/field-types.md

```diff
@@ -97,7 +97,7 @@ Currency field holds numeric value, like Item Price, Amount, etc. Currency field
 
 #### Data
 
-The data field will be a simple text field. It allows you to enter a value of up to 140 characters.
+The data field will be a simple text field. It allows you to enter a value of up to 140 characters, making this the most generic field type. To enable validations for Email or Phone Number inputs, set options to "Email" or "Phone" in the DocType settings.
```

Comment:
unclear. what do you mean by doctype settings?

Reply:
Under `Settings > DocType`

![Screenshot 2020-04-06 at 2 47 49 PM](https://user-images.githubusercontent.com/36654812/78543005-9b196300-7815-11ea-9e01-34f3ce5f8558.png)